templates/macros.html

```diff
@@ -26,7 +26,7 @@
     <td>{{ index }}</td>
     <td>{{ copy.asset_code }}</td>
     <td>{{ copy.has_cd_disk }}</td>
-    <td>{{ 'Available' if copy.available_status else 'Occupied' }}</td>
+    <td>{{ 'Available' if copy.available_status|string() == "BookStatus.RETURNED"  else 'Occupied' }}</td>
```

Comment:
How about adding third state ? If we're able to get actual book status, let's inform user about this.
E.g. `Occupied (reserved)` 
Ideally could be tooltip, but can also indicate status by some icon, whatever.